extensions/blocks/podcast-player/style.scss

```diff
@@ -58,9 +57,17 @@ $block-border-color: $dark-gray-100;
 	 * above) to apply styling for different scenarios.
 	 */
 	&.is-active {
-		color: $text-color-hover;
 		font-weight: bold;
 	}
+	&.is-active:not(.has-primary),
+	&:not(.is-active):not(.has-secondary) {
+		color: $text-color-hover;
+	}
+
+	&.is-active.has-primary .jetpack-podcast-player__episode-link,
+	&.has-secondary .jetpack-podcast-player__episode-link {
+		color: inherit;
+	}
```

Comment:
Could you please add a short comment that explains it? I think it will make it easier for the next person to grasp what's happening. It took me a bit to mentally parse it 😅

Reply:
👍

Reply:
Also, we'll have to think a nice way to define default colors, since they can be handled by CSS or through the clock properties., or both of them.